api/src/pcapi/admin/custom_views/venue_view.py

```diff
@@ -324,8 +325,27 @@ def update_model(self, new_venue_form: Form, venue: Venue) -> bool:
         for email in _get_emails_by_venue(venue):
             update_external_pro(email)
 
+        if result and venue.isVirtual == False:
```

Comment:
On évite `== False` en Python. Plutôt :
`if result and venue.isVirtual is False:`
ou plus simplement :
`if result and not venue.isVirtual:`